packages/core-plugin-api/src/extensions/extensions.tsx

```diff
@@ -51,8 +51,27 @@ export type ComponentLoader<T> =
 export function createRoutableExtension<
   T extends (props: any) => JSX.Element | null,
 >(options: {
+  /**
+   * A loader for the component that is rendered by this extension.
+   */
   component: () => Promise<T>;
+
+  /**
+   * The mount point to bind this routable extension to.
+   *
+   * If this extension is placed somewhere in the app element tree of a Backstage
+   * app, callers will be able to route to this extensions by calling,
+   * `useRouteRef` with this mount point.
```

Comment:
Could `@link` to `useRouteRef`

Reply:
Can't :<, it's overloaded and there's no way to refer to it in a way that's compatible across the various TSDoc parsers that I could find. Although it might actually be VS Code that's behind in this case. The only way to have it not trigger API reports warnings makes it should up like `:1` or smth like that in the popover.